alertmanager/router.go

```diff
@@ -0,0 +1,47 @@
+package alertmanager
+
+import (
+	"sync"
+
+	"github.com/weaveworks/mesh"
+)
+
+// Router is the interface we use for a mesh router.
```

Comment:
`Router` sounds so generic and could in Alertmanager terms easily be confused with alert routing. Maybe `GossipRouter`?